lib/internal/util.js

```diff
@@ -278,3 +278,14 @@ promisify.custom = kCustomPromisifiedSymbol;
 
 exports.promisify = promisify;
 exports.customPromisifyArgs = kCustomPromisifyArgsSymbol;
+
+exports.intrinsic = {
+  FunctionApply: Function.prototype.apply,
```

Comment:
I just realized: it's impossible to actually save a usable `Function.prototype.apply` method in JS, since one can just change `Function.prototype.apply.call` and `Function.prototype.apply.apply`… That's probably why V8 Extras supply an `uncurryThis()` function.